### PR TITLE
added the "--high_priority" argument

### PR DESCRIPTION
**Pull request by flameSla**
Increases the priority for the 'factorio' process. On Linux requires 'sudo'

### CODE REVIEW

**Comment by suisseWalter** [comment]
This causes another of by one error in the plotting. it always plots the column to the left of what is selected. 
I'm really not sure where this shift is caused.

File: benchmarker.py [hunk 1]
```diff
@@ -113,34 +124,77 @@ def run_benchmark(
...
+            result[1]
+            + {
+                "linux": " Linux",
+                "win32": " Windows",
+                "cygwin": " Cygwin",
+            }[operatingsystem_codename]
+        )
+
+    # psutil.Popen on Linux it doesn't work well with str()
+    command: list[str] = list()
```
Comment (suisseWalter) [inline benchmarker.py:143]: what is the reason for using `list()` instead of `[]`?

Comment (flameSla) [reply benchmarker.py:143]: I'm so used to writing code
replace with "[]"?

**Comment by flameSla** [comment]
>This causes another of by one error in the plotting. it always plots the column to the left of what is selected.
I'm really not sure where this shift is caused.

I didn't understand where and what is shifting
my graphs correspond to the name: "wholeUpdate.png" and in it the graph "wholeUpdate"
the numbers on the chart are also correct

**Comment by suisseWalter** [comment]
> I didn't understand where and what is shifting my graphs correspond to the name: "wholeUpdate.png" and in it the graph "wholeUpdate" the numbers on the chart are also correct

all good, there is no issue with your code, the problem is only in the consistency graphs. 
I accidentally had a old version of the master branch, where it was still working. Therefore I couldn't figure out why it was working on my master and not on your branch. but it turns out it was in my last commit where I broke it. I will push a separate fix.

File: benchmarker.py (hunk 1 above)
Comment (suisseWalter) [inline benchmarker.py:143]:
```suggestion
    command: list[str] = []
```

File: benchmarker.py [hunk 2]
```diff
@@ -113,34 +124,77 @@ def run_benchmark(
...
+            print("Map benchmarked at:")
+            print("avg = {:.3f} ms {}".format(avg, avgs_str))
+            print("{:.3f} UPS".format(ups))
+            print()
+            out: dict[str, str | float | list[str]] = dict()
+            out["version"] = version
+            out["avg"] = avg
+            out["ups"] = ups
+            out["avgs"] = avgs_str
+            filtered_output = list()
```
Comment (suisseWalter) [inline benchmarker.py:191]:
```suggestion
            filtered_output = []

File: benchmarker.py (hunk 1 above)
Comment (suisseWalter) [reply benchmarker.py:143]: for consistency with the rest of the codebase I would prefer the usage of `[]`.

**Review by suisseWalter — APPROVED** [review]
accept the two changes, list to [] and then you can merge.

One thing I've noticed during testing, but I don't think that can be fixed, is that on linux the sudo privilege causes the folders to also be created with sudo privilege which leads to them not being able to be deleted by a normal user. I don't think that this needs to be fixed. It's just a side effect.

**Pull request merged**